Makefile

```diff
@@ -26,136 +26,107 @@ GOLDFLAGS_BASE  := -X github.com/algorand/go-algorand/config.BuildNumber=$(BUILD
 GOLDFLAGS := $(GOLDFLAGS_BASE) \
 		 -X github.com/algorand/go-algorand/config.Channel=$(BUILDCHANNEL)
 
-SOURCES := $(shell cd $(SRCPATH) && \
-		go list ./... | grep -v /go-algorand/test/)
+SOURCES := $(shell go list ./... | grep -v /go-algorand/test/)
 
-UNIT_TEST_SOURCES := $(sort $(shell cd $(SRCPATH) && go list ./... | grep -v /go-algorand/test/ | grep -v /go-algorand/vendor/ ))
-E2E_TEST_SOURCES := $(shell cd $(SRCPATH)/test/e2e-go && go list ./...)
+UNIT_TEST_SOURCES := $(sort $(shell go list ./... | grep -v /go-algorand/test/ | grep -v /go-algorand/vendor/ ))
+E2E_TEST_SOURCES := $(shell cd test/e2e-go && go list ./...)
 
 default: build
 
 # tools
 
 fmt:
-	cd $(SRCPATH) && \
-		go fmt `go list ./... | grep -v /vendor/`
+	go fmt `go list ./... | grep -v /vendor/`
```

Comment:
We don't need `grep -v /vendor/` here: https://github.com/algorand/go-algorand/pull/178/files

Reply:
Hah, yeah, thanks!  That's cleaner.